Gap3/Packages3/mop.html

```diff
@@ -17,8 +17,8 @@ <h3>
   Authors
 </h3>
 <p>
-  <mixer person="Ulf Juergens" data="name"/> and
-  <mixer person="Gerhard Erich Roehrle" data="name"/>.
+  Ulf Juergens and
+  {% include namelink.html manual="Gerhard Erich Roehrle" text="Gerhard Erich Roehrle" %}.
```

Comment:
Why is this with a link?

Reply:
Every time people appeared in the addresses.yml file I used their contact information as a link.

Reply:
but Ulf Juergens isn't a link (and it wasn't before, as `data="name"` was used). yet `Gerhard Erich Roehrle` is a link now, even though it also used `data="name"` before. Why this change? I think it should stay as it was, i.e. `Gerhard Erich Roehrle` just a name

Reply:
Alright, I guess that it was just me who did it this way. I'll change it back.

Reply:
I already changed it.

Reply:
Thanks!